python/distant_vfx/jobs/ale_parse_concat.py

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env python3
+#!/usr/bin/env python
```

Comment:
If these files are executed from the commandline and you call them like `/mnt/Plugins/python3.6/distant_vfx/jobs/ale_parse_concat.py` instead of `python3 /mnt/Plugins/python3.6/distant_vfx/jobs/ale_parse_concat.py` this might turn into different functionality for you.


If that is what you want, we may want to revert this part

Reply:
I've actually been calling them via entry point scripts located outside the module - was trying to call them as you have it above but it was giving me import error headaches, for the sake of time I just added super short external entry scripts like `sort_job.py` that you see in the repo and so on.

Reply:
Ok, we'll I'll revert this change because they seem to be executable scripts and that doesn't affect me much :)